unittests/test_logging.py

```diff
@@ -3,23 +3,27 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
+import pdb
+import copy
 import logging
 import logging.handlers
 import os
 import pytest
-import sys
 import re
+import sys
 import tempfile
 import time
 import unittest
```

Comment:
Convert also the `TestLogger` class to pytest syntax

Reply:
I did only the necessary, because I want to leave that task for the #1149.